### PR TITLE
various updates to "build whl" workflow

### PR DESCRIPTION
**Pull request by andy-neuma**
SUMMARY:
* updates whl generation workflow to add testing and `testmo` integration
* add top-level generate whls workflow

TEST PLAN:
ran manually ...

### CODE REVIEW

File: .github/actions/nm-set-env/action.yml [hunk 1]
```diff
@@ -11,6 +11,7 @@ runs:
   using: composite
   steps:
   - run: |
+      echo "TORCH_CUDA_ARCH_LIST=7.0,7.5,8.0,8.6,8.9,9.0+PTX" >> $GITHUB_ENV
```
Comment (bnellnm) [inline .github/actions/nm-set-env/action.yml:14]: the vllm Dockerfile separates these with spaces.  i'm not sure how strict the format is.

Comment (bnellnm) [reply .github/actions/nm-set-env/action.yml]: seems like it might be fairly strict: https://github.com/pytorch/audio/issues/1940

Comment (andy-neuma) [reply .github/actions/nm-set-env/action.yml]: yeah, this was a typo. fixed.

**Review by bnellnm — APPROVED** [review]

File: setup.py [hunk 2]
```diff
@@ -464,10 +464,16 @@ def get_requirements() -> List[str]:
...
-                                      exclude=("neuralmagic", "neuralmagic.*",
-                                               "tests", "tests.*")),
+    license_files=('LICENSE', 'licenses/LICENSE.apache',
+                   'licenses/LICENSE.awq',
+                   'licenses/LICENSE.fastertransformer',
+                   'licenses/LICENSE.gptq', 'licenses/LICENSE.marlin',
+                   'licenses/LICENSE.punica', 'licenses/LICENSE.squeezellm',
+                   'licenses/LICENSE.tensorrtllm', 'licenses/LICENSE.vllm'),
+    packages=setuptools.find_packages(where=ROOT_DIR,
+                                      include=("vllm", "licenses", "csrc"),
```
Comment (mgoin) [inline setup.py]: do you need to include `csrc`? i would think the lib would be copied to vllm

Comment (andy-neuma) [reply setup.py]: this code is outdated

**Pull request merged**